examples/Bremsstrahlung/pymodules/energyHistogram.py

```diff
@@ -0,0 +1,227 @@
+import sys
+import os
+import inspect
+
+import numpy as np
+import matplotlib as mpl
+from matplotlib import pyplot as plt
+import h5py
+
+# prepare imports
+cmd_subfolder = os.path.realpath(
+    os.path.abspath(
+        os.path.join(
+            os.path.split(
+                inspect.getfile(
+                    inspect.currentframe()
+                )
+            )[0], ".."
+        )
+    )
+)
+if cmd_subfolder not in sys.path:
+    sys.path.insert(0, cmd_subfolder)
+
+
+# custom matplotlib settings
+params = {
```

Comment:
this should not be done in a module.
if someone changes mpl rc params, it's user/app code and not a module